### ISSUE TITLE
Inbox messages won't delete

### ISSUE THREAD

**Issue by larrymcj** [nylas/N1#1451]
Using OS X 11.11.3 with Nylas N1 Version 0.4.9-575c091 (0.4.9-575c091). No plugins installed.

When I highlight a message in my inbox and click the Trash icon at the top right, on some messages it doesn't move the message to the Trash, it just says it has changed the Tag. On other messages when I do this the messages are moved to Trash. Thanks for any help.

**Comment by bengotow** [comment]
Hi @larrymcj — thanks for filing this. It sounds like this might be related to another issue, where the thread list fails to refresh when you get it into an invalid state. If you switch away from the view (eg: to Drafts) and back, does it start working? Either way, we'll get this fixed.

**Comment by larrymcj** [comment]
Thanks for replying, Ben. Actually, moving to another view was the first thing
I tried and that didn't fix it. But then I closed N1 and restarted it again
and it hasn't happened since. Perhaps this was something related to my Mac.
Unless someone else reports it, I'd consider that the case. If I encounter it
again I'll let you know. Many thanks again.

  

Larry

> On Feb 26 2016, at 8:10 pm, Ben Gotow &lt;notifications@github.com&gt;
wrote:  

>

> Hi [@larrymcj](https://github.com/larrymcj) — thanks for filing this. It
sounds like this might be related to another issue, where the thread list
fails to refresh when you get it into an invalid state. If you switch away
from the view (eg: to Drafts) and back, does it start working? Either way,
we'll get this fixed.

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/1451#issuecomment-189545039).![](https://github.com/notifications/bea
con/AGhiWIeiVqF7xcVsZphQ52iy8sNDyAP2ks5poO79gaJpZM4HkGcK.gif)
